src/components/Stack/README.md

```diff
@@ -138,6 +140,19 @@ Use for aligning buttons or secondary content to the right edge of another eleme
 
 ---
 
+## Stack item
+
+The stack component will wrap each item in a stack item component unless an item has already been deliberately wrapped in a stack item component. Use the stack item component with the fill prop to force the item to fill the remaining horizontal space in the stack. See the [Stack where a single item fills the remaining space](#single-item-fills-remaining-space) example.
```

Comment:
@selenehinkley Would you be able to take a look at the  current wording of this and let me know what you think?

Reply:
🎉 Stack will treat multiple elements wrapped in a stack item component as one item. By default, each individual element is treated as one item. Use the fill prop on a single stack item to make it fill the rest of the available horizontal space.